_posts/2023-02-01-solidity-0.8.18-release-announcement.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.18 Release Announcement'
+date: '2023-02-01'
+author: Solidity Team
+category: Releases
+---
+
+## Introducing the newest version of the Solidity Compiler!
+We are excited to announce the latest release of the Solidity Compiler, [Solidity v0.8.18](https://github.com/ethereum/solidity/releases/tag/v0.8.18). 
+This latest version includes a range of improvements and it also introduces support for the [Paris upgrade](https://blog.ethereum.org/2022/08/24/mainnet-merge-announcement)!
+
+
+### Notable New Features
+
+#### --no-cbor-metadataCommand-Line Flag
+ A new command-line flag (`--no-cbor-metadata`) and Standard JSON option (`settings.metadata.appendCBOR: false`) to prevent compiler from appending the CBOR metadata section at the end of the bytecode. Until now, it was only possible to exclude the hash of the JSON metadata file (`--metadata-hash none`/`settings.metadata.bytecodehash: "none"`) from that section, but the bytecode would still include a small bit of CBOR data containing the version of the compiler.
+
+#### Support for Paris Hardfork
+The support for the Paris hard fork, includes: 
+    * Deprecation of global `block.difficulty` and removal of `difficulty()` in inline assembly for EVM versions >= Paris. The change is due to the renaming and semantic changes of the `difficulty` instruction introduced by [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399).
+    * Introduction of global `block.prevrandao` in Solidity and `prevrandao()` in inline assembly for EVM versions >= Paris.
+    *Note*: `block.difficulty` is planned to be removed entirely in Solidity version `0.9.0`.
+* `selfdestruct` is now considered deprecated ([EIP-6049](https://eips.ethereum.org/EIPS/eip-6049)) and the compiler will warn about its use, both in Solidity and in Yul, including inline assembly. There is currently no replacement, but its use is highly discouraged because it will eventually change its semantics and all contracts using it will be affected in some way.
```

Comment:
```suggestion
#### Deprecation of `selfdestruct`
`selfdestruct` is now considered deprecated ([EIP-6049](https://eips.ethereum.org/EIPS/eip-6049)) and the compiler will warn about its use, both in Solidity and in Yul, including inline assembly. There is currently no replacement, but its use is highly discouraged because it will eventually change its semantics and all contracts using it will be affected in some way.

```